src/commands/Canvas/comrade.ts

```diff
@@ -0,0 +1,27 @@
+import { AstraeaCommand, AstraeaCommandOptions } from '../../lib/Structures/Command'
+import { Message, MessageAttachment } from 'discord.js'
+import { ApplyOptions } from '@sapphire/decorators'
+import { Args } from '@sapphire/framework'
+import canvas from '../../lib/Canvas-SRA/requests'
+
+@ApplyOptions<AstraeaCommandOptions>({
+	name: 'comrade',
+	aliases: ['russia', 'soviet', 'russian'],
+	description: 'FOR DA MOTHER LAAAAND!'
+})
+export default class Comrade extends AstraeaCommand {
+	public async run (message: Message, args: Args): Promise<Message> {
+		let buffer
+		const mention = (await args.pickResult('user')).value
+
+		const wait = await message.channel.send('Please wait...')
+
+		buffer = await canvas('comrade', message.author.avatarURL({ format: 'png', size: 256 }))
+		if (mention)buffer = await canvas('comrade', mention.avatarURL({ format: 'png', size: 256 }))
```

Comment:
in this case, you are potentially creating a canvas that is just overwritten, you could also order these as follows

```js

if (mention) { buffer = ... }
else { buffer = ... }
```


also, please have eslint clean up before merging